frontend/src/pages/_app.tsx

```diff
@@ -35,15 +41,18 @@ class _App extends App {
 
     let currentNearNetwork, cookies, acceptedLanguages;
     if (typeof window === "undefined") {
-      currentNearNetwork = getNearNetwork(appContext.ctx.req.headers.host);
-      cookies = appContext.ctx.req.headers.cookie;
-      acceptedLanguages = appContext.ctx.req.headers["accept-language"];
+      const req = appContext.ctx.req;
+      if (req) {
+        currentNearNetwork = getNearNetwork(req.headers.host);
+        cookies = req.headers.cookie;
+        acceptedLanguages = req.headers["accept-language"];
+      }
     } else {
       currentNearNetwork = getNearNetwork(window.location.host);
     }
     return {
       currentNearNetwork,
-      ...(await App.getInitialProps({ ...appContext, currentNearNetwork })),
```

Comment:
Why did [we put](https://github.com/near/near-explorer/commit/f6e22c0b41ec1852b04b5e8165d501a2873049f2#diff-c74b166bc48fd32a6900a92b516a36c250f268be81d57463ce342172c3148ecbL25-R39) `currentNearNetwork` here? It seems like `App.getInitialProps` doesn't do anything with it and nothing seems broken after I removed it.

Reply:
I don’t see the reason either. I must have refactored something along the way and it was a leftover from some intermediate steps. Good catch!